src/super_gradients/recipes/coco2017_ssd_lite_mobilenet_v2.yaml

```diff
@@ -35,10 +35,10 @@ architecture: ssd_lite_mobilenet_v2
 
 data_loader_num_workers: 8
 
-experiment_suffix: res${dataset_params.train_image_size}
+experiment_suffix: res320
```

Comment:
are these changes intentional?

Reply:
Yes, this interpolation is not up to date (we removed all of these dataset_params.train_image_size when we got rid of datasetinterface)